Tests/PointFreeTests/AccountTests/__Snapshots__/ChangeTests/testChangeUpdateAddSeatsIndividualPlan.1.Conn.txt

```diff
@@ -5,7 +5,7 @@
   POST http://localhost:8080/account/subscription/change
   Cookie: pf_session={"userId":"00000000-0000-0000-0000-000000000000"}
 
-  lane=team&billing=monthly&quantity=4
+  lane=team&quantity=4&billing=monthly
```

Comment:
ouch hope this is deterministic.

Reply:
It's...not. New Hashable stuff. We need to fix a bunch of these by abandoning dicts or sorting somehow.

Worse is the JSONSerialization we depend on...